includes/data-stores/class-wc-product-variable-data-store-cpt.php

```diff
@@ -386,6 +386,32 @@ public function child_is_in_stock( $product ) {
 		return count( $children ) > $oufofstock_children;
 	}
 
+	/**
+	 * Are the children on backorder?
+	 *
+	 * @since 3.3.0
+	 * @param WC_Product $product Product object.
+	 * @return boolean
+	 */
+	public function children_are_on_backorder( $product ) {
```

Comment:
Seems like an ideal place to unify this with child_is_in_stock so you can check a stock status applies to all children? That way the data store just needs a single method going forward (child_is_in_stock can use this new method for bw compat).

In this case, if a child is out of stock, all are not backordered?

Reply:
> In this case, if a child is out of stock, all are not backordered?

I'm not sure what your are saying here. Can you elaborate?

Reply:
You could have a children_have_stock_status method which does both is what I'm suggesting.